src/main/resources/db/migration/V12__iat_ddl.sql

```diff
@@ -0,0 +1,23 @@
+--
+-- https://jira.fairwaytech.com/browse/IAT-3155
+--
+
+-- item history
+CREATE TABLE IF NOT EXISTS item_history (
+  id                    UUID        NOT NULL,
+  item_id               TEXT        NOT NULL CHECK (char_length(item_id)  <= 255),
+  item_commit_id        UUID,
```

Comment:
I'm not sure if we want to enforce it or not, but should this be an FK to the item table?  As in you can't enter a UUID that isn't in item?

Reply:
When importing GitLab commit records this value will be null.  The column is nullable, but I think it's possible to have a nullable column and still a FK on it.

I have been looking at the sync table and history table as independent, isolated services from the item table managed by IMS/IBMS, but since that isn't really true having an FK is reasonable, but again if the column can also be null.